package.json

```diff
@@ -47,8 +47,8 @@
     "nyc": "^14.1.1"
   },
   "dependencies": {
-    "remark-parse": "^5.0.0",
-    "unified": "^6.1.2"
+    "remark-parse": ">=5.0.0 <9.0.0",
+    "unified": ">=6.0.0"
```

Comment:
Are these version changes a necessary part of this PR? If the PR works without changing them, we'll be able to merge this as a semver-patch bugfix and release it as v2.0.1.

---

For more context in case you're interested, I've opened #175 to upgrade the parser to v7. At the very least that'll be a semver-minor change, so v2.1.0. I'm waiting for confirmation from the team that we're comfortable doing it as only a minor version bump. There's some risk that a parser change in one of its major versions will cause this processor to behave differently in a way that would require us to do a major bump to v3.0.0 here as well. Since v6, v7, and v8 are just bug fixes, I'm less concerned about noticeable breakage there. v7 fixes a bug in this processor, so to reduce risk in #175, I only went as far as needed to fix the processor bugs.

Reply:
OK, it will revert this change.